dictionaries/CallMap.php

```diff
@@ -8525,7 +8525,7 @@
 'mysqli_stmt::bind_result' => ['bool', '&w_var1'=>'', '&...w_vars='=>''],
 'mysqli_stmt::close' => ['bool'],
 'mysqli_stmt::data_seek' => ['void', 'offset'=>'int'],
-'mysqli_stmt::execute' => ['bool'],
+'mysqli_stmt::execute' => ['bool', 'params='=>'array|null'],
```

Comment:
RFC (https://wiki.php.net/rfc/mysqli_bind_in_execute) says only lists are accepted. So I think this should be `list<mixed>|null`.